hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -131,6 +131,11 @@ public final class ScmConfigKeys {
       "hdds.ratis.snapshot.threshold";
   public static final long HDDS_RATIS_SNAPSHOT_THRESHOLD_DEFAULT = 100000;
 
+  public static final String HDDS_CONTAINER_LIST_MAX_COUNT =
+      "hdds.container.list.max.count";
```

Comment:
configuration should be ozone.scm.container.list.max.count. as this is scm configuration and no relation with data nodes.